configuration/pih/scripts/global/zl.js

```diff
@@ -131,19 +131,21 @@ function setUpPhoneNumberRegex(badPhoneNumberMsg) {
 
     jq(this).change(function (e) {
       let phone = e.target.value;
-      if (phone.match(phoneNumberPattern().pattern1) || phone.match(phoneNumberPattern().pattern2) || phone.match(phoneNumberPattern().pattern3)) {
+      if (phoneNumberRegex(phone)) {
```

Comment:
I just pulled this check out into it's own function on that I could use it in the above code as well.